extensions/positron-python/resources/walkthrough/panes.md

```diff
@@ -0,0 +1,17 @@
+Positron has multiple panes, built to support the way data scientists and Python developers work:
+
+- Run Python code interactively in [the Console](command:workbench.action.positronConsole.focusConsole).
+- Use the [File Explorer](command:workbench.files.action.focusFilesExplorer) to navigate your files and projects.
+- Use the [Outline](command:outline.focus) to see the structure of your Python files, including code cells, functions, and classes.
```

Comment:
Do you want to include these, given that they are the same as in in VS Code? And similarly for the terminal?